### DISCUSSION TITLE
Task status breakdown of projects in projects list

### DISCUSSION THREAD

**Discussion by ramtinabadi** [community/community#67169] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

We use projects to manage the issues related to a specific release across multiple repos. 
It would be very helpful to see a status breakdown of the tasks or progress of each project, similar to milestones. Since we are tracking issues across multiple repos, we cannot use milestones efficiently but having a similar overview of the progress of projects would be very useful.

I propose placing a component similar to the language breakdown of repos (see the first image) in the right side of the project card (see the second image). This component should give a breakdown of the statuses (To Do, In Progress, Done, etc.).

With this addition, you would immediately get a general overview of the progress of all projects, similar to that of milestones. 

## 1
<img width="326" alt="Screenshot 2023-09-15 at 1 36 29 PM" src="https://github.com/community/community/assets/17789994/57db29de-c74a-454b-827d-d3fc9f27958a">

## 2
<img width="951" alt="Screenshot 2023-09-15 at 1 37 24 PM" src="https://github.com/community/community/assets/17789994/0034e2fe-50c6-4324-a38a-ef2f04dfbae5">

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by RichardForshaw** [comment]
I would like to second this.

When managing multiple projects and repositories, it would be very helpful to see an overview/dashboard of projects' milestone progress and/or status counts (given any project is likely to only be working on one milestone at a time).

This currently only seems to be possible by setting up a board with grouping and slices in each project, which works OK but is still only an overview for a single project.

**Comment by naveen-98** [comment]
Hello!

As an alternative solution, consider the following:

1. **Use Project Automation:**
   - Leverage GitHub Actions or other automation tools to update a custom field in your project card based on the status of issues. For example, you could add a comment with a specific keyword or use labels to signify the status, and then have a workflow update a custom field.

2. **Custom Dashboards:**
   - Create custom dashboards using GitHub API or third-party tools that aggregate information from multiple repositories and present a summary of project progress. Services like GitHub GraphQL API can be utilized to gather detailed information about issues across repositories.

3. **Third-Party Tools:**
   - Explore third-party project management tools that integrate with GitHub and provide enhanced reporting and progress tracking features. Tools like ZenHub, Waffle.io, or others might offer the functionality you're looking for.

4. **GitHub Discussions:**
   - Visit the GitHub Discussions for GitHub features and share your idea/request. GitHub's team actively monitors and considers community feedback when planning new features. Users can discuss and upvote feature requests.

**Reply by ramtinabadi** [reply]
The solutions you have provided, even though credible, do not make the lives of the users easier.
The user should not spend days dealing with APIs and/or third-party services just to have an overview of their projects.
